Accepted-RFCs/RFC-011-news-announcements.md

```diff
@@ -0,0 +1,165 @@
+
+Heartbeats could also possibly showcase an individual contributor, like a "meet the team" sort of vibe. This has been declined as a blog post topic, as that doesn't fit the purpose of the blog, but it could fit here at the tail end of heartbeat posts.
+
+(That also gives those individual contributors featured a different sort of call to action on publicizing their related post, as friends/colleagues of that contributor may be interest in the post for relationship reasons, then find the other content or project in general interesting enough to follow on its own.)
+
+#### What this isn't
+
+This wouldn't be a place for internal communications, meeting notes, etc. 
+
+This also wouldn't be a "second blog" or similar.
+
+### Why not use the blog?
+
+The blog is about "docs advocacy & education," so if peopple subscribe to the blog/look out for new posts, they should expect material that's education, not that's purely about our group marketing.
+
+The other elements of the blog, like its particular formatting/visual branding, focus on author, everything about CC licensing, etc. do not and should not apply to this content bucket.
+
+### Specifics
+
+The main page should show the most recent post, so when one clicks on "News" in the header, they don't have to click on a second link to see what's the most recent.
+
+The sidebar would have the most recent 5 or so posts, with a "see all" that would just show a list with short versions, akin to what is seen in the [Write the Docs newsletter archive page](https://www.writethedocs.org/blog/archive/tag/newsletter/).
+
+The posts should be short: 500 words or so. Something that would take on average less than 5 minutes to read. We may deviate from this as needed, but the core concept is that our newsletters shouldn't take significant resources to craft and review, and we shouldn't demand significant time expenses from our audience in this manner.
+
+Unlike how we handle the blog, authors wouldn't link to "about the author" pages as we do in the blog—and in fact we may decide there are no listed individual authors, to keep the news to appear to always be from the community.
+
+Unlike how we handle the blog, there wouldn't be any commenting feature. 
+
+Unlike how we handle the blog, there wouldn't be any banner or hero images.
+
+Because news is timely, the individual post URLs would use the date rather than a title.
+
+### Why not use the blog?
+
+Two reasons:  First, it would split the purpose and publishing resources of the blog. The blog's purpose is supporting docs advocacy and education.
+
+Second, while the framework might be similar, it isn't exactly the same. So we can reuse a lot of the work on the blog, but it has different content needs.
```

Comment:
I see a different 'content type' for the markdown files, and new layouts for this section of the site.  Something like a 'twitter feed' with a card for each one.

Reply:
Totally. I'll leave the markdown implementation as a post-RFC phase, since I'm sure like with the blog, we'll discover nuances as we build it out. But I'm def picking up what you're putting down.